test/cmd/basicresources.sh

```diff
@@ -26,7 +26,8 @@ os::test::junit::declare_suite_start "cmd/basicresources/versionreporting"
 os::build::get_version_vars
 os_git_regex="$( escape_regex "${OS_GIT_VERSION%%-*}" )"
 kube_git_regex="$( escape_regex "${KUBE_GIT_VERSION%%-*}" )"
-etcd_git_regex="$( escape_regex "${ETCD_GIT_VERSION%%-*}" )"
+etcd_version="$(echo "${ETCD_GIT_VERSION}" | sed -E "s/\-.*//g" | sed -E "s/v//")"
```

Comment:
Can we achieve this transform without hierloglyphics? Nobody wants to see a string of `seds`